src/applications/personalization/account/components/TermsAndConditions.jsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
 import recordEvent from '../../../../platform/monitoring/record-event';
+import isBrandConsolidationEnabled from '../../../../platform/brand-consolidation/feature-flag';
+
+const propertyName = isBrandConsolidationEnabled() ? 'VA.gov' : 'Vets.gov';
```

Comment:
should we create a `getPropertyName` helper?

Reply:
I figured it would be easier to refactor out by having references to `isBrandConsolidationEnabled` everywhere it's used. should only have to search that string to find everything that needs to be deleted when `vets.gov` is deprecated.